.cicd/scripts/srw_ftest.sh

```diff
@@ -95,8 +92,10 @@ module load wflow_${platform,,}
 conda activate srw_app
 set -e -u
 
-# Adjust for strict limitation of stack size 
-sed "s|ulimit -s unlimited;|ulimit -S -s unlimited;|" -i ${workspace}/ush/machine/hera.yaml
```

Comment:
I also removed this line. If Hera needs this syntax, we should make the change in the hera machine file. If also doesn't make sense to change _only_ this machine file when this script runs on other platforms.